pages/sign-up.js

```diff
@@ -17,10 +25,25 @@ const SignUp = () => {
       confirmPassword: yup
         .string()
         .required('Confirm password is required')
-        .oneOf([yup.ref('password')], "Password's not match")
+        .oneOf([yup.ref('password')], "Password's not match"),
+      acceptTerms: yup.boolean().oneOf([true], 'Must Accept Terms and Conditions'),
+      agreeContact: yup.boolean(),
+      acceptMulti: yup
```

Comment:
Since we have confirmed that the Checkbox group is working as expected. Can we remove this as it is not needed in the code.

Reply:
Think only the ` acceptMulti: yup` needs to be removed here since the checkbox group is not needed in the design yet `  acceptTerms: yup.` and `agreeContact: yup` is used to validate the `checkbox component`

Reply:
Thank you for your response on this, however, you repeated what I mean by my comment 😄 . 

> Can we remove this as it is not needed in the code.

The above statement is pointing at line 31 (since the comment is made on that line). I was referring only to the `acceptMulti` that was why the comment was made on line `31`.